geomesa-accumulo/geomesa-accumulo-tools/bin/install-hadoop-accumulo.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#! /usr/bin/env bash
```

Comment:
That's a style question. I always have but both work, it originally had the space.

> To take advantage of this wonderful opportunity, put
>
>#! /bin/sh
>
>at the left margin of the first line of your shell scripts. Blanks after ! are OK.  Use a complete pathname (no search is done). At the moment the whole line is restricted to 16 characters but
this limit will be raised.

[#! Announcement Letter, Dennis Ritchie](https://en.wikipedia.org/wiki/Shebang_(Unix)#History)